code/__DEFINES/stat.dm

```diff
@@ -13,10 +13,12 @@
 #define MAX_SATIETY 600
 
 // bitflags for machine stat variable
-#define BROKEN	(1<<0)
-#define NOPOWER	(1<<1)
-#define MAINT	(1<<2)		// under maintaince
-#define EMPED	(1<<3)		// temporary broken by EMP pulse
+#define BROKEN		(1<<0)
+#define NOPOWER		(1<<1)
+#define MAINT		(1<<2)	// under maintenance
+#define EMPED		(1<<3)	// temporary broken by EMP pulse
+#define OVERHEATED	(1<<4)	// overheated
+#define TURNED_OFF	(1<<5)	// turned off
```

Comment:
Please use `//! ` so that we can read the comments by hovering a cursor on them.

Reply:
done